maintainers/maintainer-list.nix

```diff
@@ -85,6 +85,15 @@
     github = "baldo";
     name = "Andreas Baldeau";
   };
+  abbe = {
```

Comment:
Just curious — the maintainer nickname doesn't coincide with anything in the maintainer entry. Should we add some fields to the schema? (Is it your IRC nickname?)

Reply:
Yes, it's.